Gemfile.lock

```diff
@@ -2,71 +2,131 @@ GEM
   remote: http://rubygems.org/
```

Comment:
I see... it's just always the first step for me to delete Gemfile.lock when working with packaged rubygems/jekyll :)